### ISSUE TITLE
关于图卷积操作部分内容

### ISSUE THREAD

**Issue by Ricardo-Ping** [HuilinChenJN/DMRL-PyTorch#1]
为什么在代码实现过程中会有图卷积的操作呢？
        for layer in range(self.n_layers):
            if self.A_split:
                temp_emb = []
                for f in range(len(g_droped)):
                    temp_emb.append(torch.sparse.mm(g_droped[f], all_emb))
                side_emb = torch.cat(temp_emb, dim=0)
                all_emb = side_emb
            else:
                all_emb = torch.sparse.mm(g_droped, all_emb)
            embs.append(all_emb)
        embs = torch.stack(embs, dim=1)
        light_out = torch.mean(embs, dim=1)
        users, items = torch.split(light_out, [self.n_users, self.n_items])
